dev/tools/proto-to-mapper/Makefile

```diff
@@ -11,7 +11,7 @@ generate-pb: install-protoc-linux
 		-I ./third_party/googleapis/ \
 		-I ../../../mockgcp/apis \
 		../../../mockgcp/apis/mockgcp/cloud/networkconnectivity/*/*.proto \
-		../../../mockgcp/apis/google/apps/cloudidentity/groups/*/*.proto \
```

Comment:
is the metadata file changing. Make sure you send that to walter. It may be ok to add that as part of this PR.